chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -144,6 +144,7 @@
 <!ENTITY zotero.preferences.proxies.desc_after_link 		"for more information.">
 <!ENTITY zotero.preferences.proxies.transparent				"Enable proxy redirection">
 <!ENTITY zotero.preferences.proxies.autoRecognize			"Automatically recognize proxied resources">
+<!ENTITY zotero.preferences.proxies.showRedirectNotification			"Show notification when automatically redirecting through a proxy">
```

Comment:
We can drop the "automatically" here. That's just how the feature works, but it's not relevant to this option.